specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/examples/SearchIndexGetAutocomplete.json

```diff
@@ -6,7 +6,13 @@
         "api-version": "2017-11-11-Preview",
         "autocompleteMode": "oneTerm",
         "search": "washington medic",
-        "suggesterName": "sg"
+        "suggesterName": "sg",
+        "fuzzy": false,
+        "highlightPostTag": "</em>",
+        "highlightPreTag": "<em>",
+        "minimumCoverage": 80,
+        "searchFields": ["title", "description"],
```

Comment:
`searchFields` is an `IList` in the C# SDK, but it's still a `string` as far as Swagger is concerned, right?